roles/config/tasks/activation_key.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: "Create generic_key activation key"
+  theforeman.foreman.katello_activation_key:
+    username: "{{ lookup('env','FOREMAN_USER') }}"
+    password: "{{ lookup('env','FOREMAN_PASSWORD') }}"
+    server_url: "https://{{ ansible_fqdn }}"
+    name: "generic_key"
+    organization: "{{ foreman_initial_organization }}"
```

Comment:
To fit OASIS naming patterns this should be `satellite_config_initial_organization` or similar